### PR TITLE
feat: add catchup 103 summary

### PR DESCRIPTION
**Pull request by SirusCodes**

### CODE REVIEW

**Comment by HarshKapadia2** [comment]
Please squash and merge after review. Thank you!

File: summary/sessions/103/content.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Date: 29-10-2022
+
+Duration: 3 hrs 39 mins
+
+==== Topics Discussed
+
+* link:https://twitter.com/AnxKhn[Anas Khan^] asked link:https://twitter.com/furtado_jaden[Jaden Furtado^] how one can submit IP address in link:https://github.com/JadenFurtado/geoAbuseIPDB[AbuseIP DB^].
+    ** AbuseIP DB is a DB of IP address that tried to attack a system in past.
```
Comment (tusharnankani) [inline summary/sessions/103/content.adoc:8]: - that has* tried 
- in the* past

File: summary/sessions/103/content.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] pointed him to the the link:https://www.abuseipdb.com/report[reporting page^] on site.
```
Comment (tusharnankani) [inline summary/sessions/103/content.adoc:9]: [reporting page of Abuse IP DB^].

File: summary/sessions/103/content.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    ** link:https://dev.to/byteslash/deta-space-what-is-it-and-why-is-it-the-future-52fj[More on Deta Space^]
+* link:https://kubernetes.io[Kubernetes^] (K8s)
+    ** link:https://twitter.com/rishit_dagli[Rishit Dagli^] talked about his experience at link:https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america[KubeCon and CloudNativeCon North America 2022^].
+    ** Rishit talked about the cost of running the K8s project and the improvements to the project's registry in upcoming versions to make it more efficient, so that all requests for packages are not made to the main K8s registry.
+    ** Rishit also told us about Release Engineering at K8s which handles all release procedures such as identifying flaky tests, deciding how to release a particular version, in what manner (phased or all at once), etc.
+    ** Rishit shared tests of some of the K8s projects
+        *** link:https://testgrid.k8s.io[testgrid.k8s.io^]
+        *** link:https://github.com/kubernetes/test-infra[github.com/kubernetes/test-infra^]
```
Comment (tusharnankani) [inline summary/sessions/103/content.adoc:26]: Why is the parent link of K8s?

Comment (SirusCodes) [reply summary/sessions/103/content.adoc:26]: ig it's because the show section is talking about k8s

Comment (tusharnankani) [reply summary/sessions/103/content.adoc]: Oh okay.

File: summary/sessions/103/content.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
+Date: 29-10-2022
+
+Duration: 3 hrs 39 mins
+
+==== Topics Discussed
+
+* link:https://twitter.com/AnxKhn[Anas Khan^] asked link:https://twitter.com/furtado_jaden[Jaden Furtado^] how one can submit IP address in link:https://github.com/JadenFurtado/geoAbuseIPDB[AbuseIP DB^].
+    ** AbuseIP DB is a DB of IP addresses that have tried to attack a system in the past.
+    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] pointed him to the the link:https://www.abuseipdb.com/report[reporting page of AbuseIP DB^].
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:9]: Jaden shared ... for addressing Anas's question.

File: summary/sessions/103/content.adoc [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* link:https://twitter.com/SirusTweets[Darshan Rander^] shared his recent contributions to link:https://github.com/microsoft/winget-cli/pull/2650[winget-cli] and link:https://github.com/fluttercommunity/plus_plugins/pull/1295[plus_plugins].
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:10]: Should it be added in Projects showcased section.

File: summary/sessions/103/content.adoc [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    ** In the link:https://github.com/microsoft/winget-cli/pull/2650[winget-cli] he created a spec for a selector while installing a package if there are multiple matches.
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:11]: He created a winget-cli spec for a selector while ...

File: summary/sessions/103/content.adoc [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    ** In the link:https://github.com/fluttercommunity/plus_plugins/pull/1295[plus_plugins] he replaced link:https://pub.dev/packages/url_launcher[url_launcher (Flutter plugin)] with it's platform specific versions.
+* link:https://twitter.com/SirusTweets[Darshan Rander^] asked link:https://twitter.com/_SharmaHimanshu[Himanshu Sharma^] when you should clean your laptop from inside.
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:13]: Merge the two points.

File: summary/sessions/103/content.adoc [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    ** link:https://twitter.com/_SharmaHimanshu[Himanshu Sharma^] suggested doing it once in 6 months.
+* link:https://twitter.com/kaku_jay[Jay Kaku^] shared the link:https://research.ibm.com/blog/ibm-artificial-intelligence-unit-aiu[blog] of IBM releasing a new chip for ML training.
+    ** link:https://twitter.com/SirusTweets[Darshan Rander^] asked what is the difference between NVIDIA's GPU, Google's Tensor and this chip. To which link:https://twitter.com/kaku_jay[Jay Kaku^] said the use cases of them are very different. NVIDIA is general, Tensor is for running the model and this one is for training them.
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:16]: Darshan and Jay talked about the differences between ...

File: summary/sessions/103/content.adoc [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+* link:https://twitter.com/kaku_jay[Jay Kaku^] shared a link:https://github.com/IBM/aihwkit[IBM's Analog Hardware Acceleration Kit^] a project for AI development.
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:17]: Shouldn't it be a sub-point for the previous point?

File: summary/sessions/103/content.adoc [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+* link:https://twitter.com/harshgkapadia[Harsh Kapadia^], link:https://twitter.com/SirusTweets[Darshan Rander^] and link:https://twitter.com/adityaoberai1[Aditya Oberai^] talked about link:https://deta.space[deta.space^] and what is the future of it.
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:18]: H, D, and A talked about Deta and its future.

File: summary/sessions/103/content.adoc [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+    ** link:https://dev.to/byteslash/deta-space-what-is-it-and-why-is-it-the-future-52fj[More on Deta Space^]
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:19]: Combine with previous point?

File: summary/sessions/103/content.adoc [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+* link:https://kubernetes.io[Kubernetes^] (K8s)
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:20]: Context?

File: summary/sessions/103/content.adoc (hunk 5 above)
Comment (SirusCodes) [reply summary/sessions/103/content.adoc:10]: This is not a project I'm just sharing what I did in the week.

File: summary/sessions/103/content.adoc (hunk 12 above)
Comment (PranavDani) [inline summary/sessions/103/content.adoc:20]: We talked about K8s and experiences?

File: summary/sessions/103/content.adoc [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] told us some of the matters that have to be decided upon when a company acquires another company and how complex they are.
+    ** User migration from one service to another
+    ** Codebase linting, policies and styling consistencies
+    ** Branding decisions and alignment
+    ** Service Level Agreement (SLA) changes
+    ** Business goals re-alignment
+    ** Independence level of the company getting acquired
+    ** Transitioning (fitting) employees, laying come employees off, handling employee stock conversions, deciding salaries, etc.
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] talked about the hostile takeover of Twitter, how the firing of Twitter higher-ups is not related to Recession, corporate buyouts and link:https://corporatefinanceinstitute.com/resources/knowledge/deals/golden-parachute[Golden Parachutes^].
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] and link:https://twitter.com/harshgkapadia[Harsh Kapadia^] talked about how we all are too harsh on ourselves and how we need to understand ourselves better.
```
Comment (PranavDani) [inline summary/sessions/103/content.adoc:36]: ❤️‍🔥

File: summary/sessions/103/content.adoc (hunk 8 above)
Comment (SirusCodes) [reply summary/sessions/103/content.adoc:16]: I feel this is fine

Comment (tusharnankani) [reply summary/sessions/103/content.adoc:16]: So, when we say — Darshan asked what is the difference?
It ends with a question mark. 

Potential rephrases:

> what Pranav wrote

Darshan asked what the difference is. 

Darshan asked the difference.

**Pull request merged**